widget/charts/src/components/BarChart/BarChart.constants.ts

```diff
@@ -0,0 +1,10 @@
+export const DEFAULT_MARGIN = { top: 40, right: 0, bottom: 0, left: 20 };
+
+export const SEVEN_DAYS = 7;
```

Comment:
The logic of the days should not be included in the chart component.

Reply:
fixed